src/registerDynamicSymbol.c

```diff
@@ -0,0 +1,22 @@
+// RegisteringDynamic Symbols
+
+#include <R.h>
+#include <Rinternals.h>
+#include <R_ext/Rdynload.h>
+#include "has_topotoolbox.h"
+
+static const R_CMethodDef cMethods[] = {
+   {"has_topotoolbox", (DL_FUNC) &has_topotoolbox,1},
+   NULL,
```

Comment:
There is also a compiler warning here because `gcc` expects this `NULL` to be an `R_CMethodDef` rather than just a null pointer. It doesn't really seem to affect anything at the moment, but we can at least remove the warning by replacing `NULL` with `{NULL,NULL,0,NULL}`, This is consistent with the examples in [Writing R Extensions](https://cran.rstudio.com/doc/manuals/r-devel/R-exts.html#Registering-native-routines).